src/code42cli/cmds/alerts.py

```diff
@@ -195,17 +222,46 @@ def search(
         formatter=formatter,
         force_pager=include_all,
     )
-    extractor = _get_alert_extractor(cli_state.sdk, handlers)
-    extractor.use_or_query = or_query
-    if advanced_query:
-        extractor.extract_advanced(advanced_query)
-    else:
-        if begin or end:
-            cli_state.search_filters.append(
-                ext.create_time_range_filter(f.DateObserved, begin, end)
-            )
-        extractor.extract(*cli_state.search_filters)
-    if handlers.TOTAL_EVENTS == 0 and not errors.ERRORED:
+    _call_extractor(cli_state, handlers, begin, end, or_query, advanced_query, **kwargs)
+    if not handlers.TOTAL_EVENTS and not errors.ERRORED:
+        echo("No results found.")
+
+
+@alerts.command()
+@alert_options
+@search_options
+@click.option(
+    "--or-query", is_flag=True, cls=searchopt.AdvancedQueryAndSavedSearchIncompatible
+)
+@opt.sdk_options()
+@server_options
+@click.option(
+    "--include-all",
+    default=False,
+    is_flag=True,
+    help="Display simple properties of the primary level of the nested response.",
+)
+@send_to_format_options
+def send_to(
+    cli_state,
+    format,
+    hostname,
+    protocol,
+    begin,
+    end,
+    advanced_query,
+    use_checkpoint,
+    or_query,
+    **kwargs
+):
+    """Send alerts to the given server address."""
+    logger = get_logger_for_server(hostname, protocol, format)
+    cursor = _get_alert_cursor_store(cli_state.profile.name) if use_checkpoint else None
+    handlers = ext.create_send_to_handlers(
+        cli_state.sdk, AlertExtractor, cursor, use_checkpoint, logger,
+    )
+    _call_extractor(cli_state, handlers, begin, end, or_query, advanced_query, **kwargs)
+    if not handlers.TOTAL_EVENTS and not errors.ERRORED:
```

Comment:
NOT A BLOCKER, but this could also be a separate shared method like `_handle_if_no_events()`

Reply:
I would like to make this method as below, should it be in `util` or if it is specific to search then shall I create a new `util` in `search`? This is so that it can apply both to alerts and security-data. 

```
def _handle_no_events(no_events:bool):
      if no_events:
          echo("No results found.")
```


and then it will be called as `_handle_no_events(not handlers.TOTAL_EVENTS and not errors.ERRORED)`

Reply:
I'd put it in `cmds.search.extraction` , which is like the util for only search related things, but yeah that looks like a good implementation

Reply:
Done.